lib/task.js

```diff
@@ -109,10 +109,26 @@ function factory(
                 Task.configCache.apiServerPort
             );
         }
+        var fileServerUri;
+        if (_.has(Task.configCache, 'fileServerAddress') &&
+            _.has(Task.configCache, 'fileServerPort') &&
+            _.has(Task.configCache, 'fileServerPath')) {
```

Comment:
I would suggest not making fileServerPath a mandatory field. If user meant to use the root directory of the file server, they would got a chance of not specifying the fileServerPath.

This is a non blocking anyway.